clippy_lints/src/neg_multiply.rs

```diff
@@ -49,8 +54,21 @@ fn check_mul(cx: &LateContext<'_>, span: Span, lit: &Expr<'_>, exp: &Expr<'_>) {
         if let ExprKind::Lit(ref l) = lit.kind;
         if consts::lit_to_constant(&l.node, cx.typeck_results().expr_ty_opt(lit)) == Constant::Int(1);
         if cx.typeck_results().expr_ty(exp).is_integral();
+        if let ExprKind::Path(QPath::Resolved(_, var_path)) = exp.kind;
+
         then {
-            span_lint(cx, NEG_MULTIPLY, span, "negation by multiplying with `-1`");
+            let var_name = var_path.segments[0].ident.name.as_str();
+            let suggestion = format!("-{var}",var=var_name);
+            let applicability = Applicability::MachineApplicable;
+            span_lint_and_sugg(
+                    cx,
+                    NEG_MULTIPLY,
+                    span,
+                    "this `multiplication with -1` can be written more succinctly",
```

Comment:
```suggestion
                    "this multiplication with -1 can be written more succinctly",
```
It looks strange to me to put "multiplication with -1" in grave accents. Is there a reason for that? I would understand if it were `* -1` or something similar that's actually code.
The old string (above) used grave accents only for the number `-1` (which can be code):
```diff
-             span_lint(cx, NEG_MULTIPLY, span, "negation by multiplying with `-1`");
```
Maybe you meant to do that instead, if at all?

Reply:
Hi @r00ster91 , 
Yes you are right 👍 I will fix it.